tools/agent-ctl/src/main.rs

```diff
@@ -156,8 +156,10 @@ fn connect(name: &str, global_args: clap::ArgMatches) -> Result<()> {
             .collect();
```

Comment:
Having two commits immediately after each other with identical one-line summaries is probably not a great idea.